src/ansible.rs

```diff
@@ -198,6 +296,31 @@ impl AnsibleRunner {
         )?;
         Ok(())
     }
+
+    fn get_inventory_path(&self, inventory_type: AnsibleInventoryType) -> PathBuf {
```

Comment:
Tiny nit, but we can take in `&AnsibleInventoryType` here to avoid writing `.clone()` inside loops (as we have done above).